package.json

```diff
@@ -11,16 +11,17 @@
   },
   "dependencies": {
     "@netlify/functions": "latest",
+    "@remix-run/netlify-edge": "^1.0.0",
```

Comment:
this is under “ experimental-netlify-edge” actually, and will be versioned with our other adapters